src/api/User/index.ts

```diff
@@ -61,7 +63,19 @@ export class UserAPI extends Request {
   public searchUsers(this: Request<SearchUserResponse>, params: ISearchUsersParams) {
     return this.request({
       errors: [UNAUTHORIZED, FORBIDDEN, DATA_SOURCE_UNAVAILABLE],
-      url: '/users',
+      url: '/admin/users',
+      method: 'GET',
+      params: params
+    });
+  }
+
+  /**
+   * Get all the users or filter them by username, e-mail or group id.
+   */
+  public getUserList(this: Request<GetUserListResponse>, params: IGetUserListParams) {
```

Comment:
what do you think about naming this one "searchUsers", and renaming the version for admins "searchUsersFull"?
That way, the names are consistent with the fact that both are for searching, but one returns simple users and the other one full users with more details.